packages/connect-webextension/package.json

```diff
@@ -48,7 +48,6 @@
         "@babel/preset-typescript": "^7.23.3",
         "@trezor/node-utils": "workspace:*",
         "@trezor/trezor-user-env-link": "workspace:*",
-        "@types/chrome": "^0.0.260",
```

Comment:
you sure it is not used?

Reply:
my vscode sources types for this line from @types/chrome https://github.com/trezor/trezor-suite/blob/develop/packages/connect-web/src/channels/serviceworker-window.ts#L15

Reply:
returned back